README.md

```diff
@@ -122,51 +88,27 @@ DOCKER_API_VERSION=1.23
 export DOCKER_API_VERSION
 ```
 
-For convenience, a script called "run.sh" is provided which can be edited to specify the desired options.
-
+## Usage
 
-Run Container
-------------
-The best way to experiment with the driver is to run it as a container.
-
-A pre-built docker image can be pulled from Docker Hub using the following command:
-```
-docker pull infoblox/ipam-driver
-```
-
-After successfully pulling the image, you use the ```docker run``` command to run the driver. For exampe:
-```
-docker run -e DOCKER_API_VERSION=1.22 -v /var/run:/var/run -v /run/docker:/run/docker infoblox/ipam-driver --grid-host=192.168.124.200 --wapi-username=cloudadmin --wapi-password=cloudadmin --local-view=local_view --local-network-container="192.168.0.0/20,192.169.0.0/22" --local-prefix-length=25 --global-view=global_view --global-network-container="172.18.0.0/16" --global-prefix-length=24
-```
-
-Note that the -v options are necessary to provide the container access to the specified directories on the
-host file system.
-
-For convenience, a script called "run-container.sh" is provided.
-
-Usage
------
 To start using the driver, a docker network needs to be created specifying the driver using the --ipam-driver option:
 ```
-sudo docker network create --ipam-driver=infoblox priv-net
+$ docker network create --ipam-driver=infoblox/ipam-plugin:v1.1.0 priv-net
```

Comment:
Can you create two instances of the same driver with different configurations and different driver names? Or are we locked into one now with the new API?